### DISCUSSION TITLE
Integrating with MediaLibrary

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [spatie/laravel-settings#185] [Q&A] [answered]
Hi devs,
I got a question in mind, how to extend Laravel Settings to be used with Spatie Media Library. 🥺

**Comment by rubenvanassche** [comment, marked as answer]
Technically probably yes, but I wouldn't recommend it since settings is more for global settings not tied to a model.

**Reply by hamzaelmaghari** [reply]
Thanks,
Your response was valuable for me.
